src/include/parser/create_statement.h

```diff
@@ -225,6 +225,117 @@ class CreateStatement : public TableRefStatement {
 
   virtual void Accept(SqlNodeVisitor *v) override { v->Visit(this); }
 
+  const std::string GetInfo(int num_indent) const {
```

Comment:
I would this method private.

Reply:
I plan to do this together with the phase of making all member variables private.